stories/topics/ref-aap-gcp-install-prerequisites.adoc

```diff
@@ -24,4 +24,90 @@ You must have a working knowledge of the following aspects of {GCP}:
 ** GCP Ops agent
 * SSH
 
+In addition to this, your GCP account must have the permissions listed below to successfully create an {PlatformNameShort} deployment on _{GCP}_.
```

Comment:
In addition, your

Reply:
Yeah that makes sense! Made the updates you requested